src/main/java/mate/jdbc/dao/impl/DriverDaoImpl.java

```diff
@@ -19,8 +20,10 @@ public class DriverDaoImpl implements DriverDao {
     public Driver create(Driver driver) {
         String query = "INSERT INTO drivers (name, license_number) VALUES (?, ?)";
         try (Connection connection = ConnectionUtil.getConnection();
-                PreparedStatement statement = connection.prepareStatement(query,
-                        Statement.RETURN_GENERATED_KEYS)) {
+                PreparedStatement statement = connection.prepareStatement(
+                        query,
+                        Statement.RETURN_GENERATED_KEYS
+                )) {
```

Comment:
it's better not to change style in already implemented part

Reply:
mb was applied autoFormatting for some misstake.
+ Changed to previous look.